tutorials/nservicebus-101/lesson-2/Snippets/Core_6/ComplexCommand.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using NServiceBus;
+
+namespace Core_6
+{
+    #region ComplexCommand
+
+    public class DoSomethingComplex :
+        ICommand
+    {
+        public int SomeId { get; set; }
+        public ChildClass ChildStuff { get; set; }
```

Comment:
I have info in the commentary about messages being a contract between two endpoints, so to keep messages as slim as possible.

I had originally said POCO but someone pointed out that if you implement an interface it's not really a POCO so I had to take that out.